pkg/verification/verification/ipxe.efi.cfg

```diff
@@ -0,0 +1,30 @@
+#!ipxe
+# dhcp
+#
+# Uncomment ntp lines for devices without RTC (RPI for example)
+# echo Getting the current time from ntp...
+# :retry_ntp
+# ntp pool.ntp.org || goto retry_ntp
+#
+# you may want to add the following to the kernel command line arguments:
+#   * eve_install_disk=XXX (e.g. XXX=mmcblk0)
```

Comment:
eve_install_* parameters are not available for verification image, right? So these comments can be removed...